examples/amass_human_poses/README.md

```diff
@@ -0,0 +1,116 @@
+
+# AMASS Dataset
+<p align="center">
+<img src="rendering.png" alt="Front readme image" width=400>
+</p>
+
+AMASS (Archive of Motion Capture as Surface Shapes) is a large database of human motion unifying different optical marker-based motion capture datasets by representing them within a common framework and parameterization. AMASS is readily useful for generating training data for deep learning.
+
+## Get The Dataset
+As you are required to agree to the Use Of Terms for this dataset, we can not provide a download script.
+
+However, we will give you a step by step explanation on how to get access.
+
+1. Visit the following websites, sign up as user after you agree to the terms of use.
```

Comment:
Do you need to sign in on each of those pages individually or once for all of them? This is a bit unclear here.

Reply:
yes, on each one of them. The first website is for dataset access and the next two are for the parametric model which generates the mesh file from data

Reply:
Did you change this also in the README?